bundles/af-core/src/test/resources/schema/0.12.1/adaptive-form.schema.json

```diff
@@ -732,6 +732,7 @@
             "properties",
             "readOnly",
             "rules",
+            "fileInputDragDropText",
```

Comment:
You don't need to change the schema. Ideally, the gh action should have flagged this change

Reply:
Removed the schema change.